images/linux/ubuntu2004.json

```diff
@@ -293,10 +293,9 @@
             "environment_vars": [
                 "METADATA_FILE={{user `metadata_file`}}",
                 "HELPER_SCRIPTS={{user `helper_script_folder`}}",
-                "GO_VERSIONS={{user `go_versions`}}",
-                "GO_DEFAULT={{user `go_default`}}"
+                "INSTALLER_SCRIPT_FOLDER={{user `installer_script_folder`}}"
```

Comment:
is it ok that for Ubuntu 18 and 16 you are installing toolset befor `go.sh` and but vice versa for Ubuntu 20?

Reply:
Moved go.sh according to ubuntu1804 and ubuntu1804